diff --git a/Makefile b/Makefile
index 79d56d7..71a6635 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 # For development only
-KEYDOGGER_KEYBOARD=/dev/input/event2
+KEYDOGGER_KEYBOARD=/dev/input/event6
 
 DEPENDENCIES=wl-copy wl-paste
 DAEMON_NAME=keydoggerd
@@ -7,7 +7,7 @@ PREFIX=/usr/local/bin/
 CALLGRIND_FILE=benchmark.out
 
 .PHONY: build
-build: keydogger.h keydogger.c dependency-check
+build: keydogger.h keydogger.c
 	gcc -DDEBUG_MODE=0 -DDAEMON_NAME=\"$(DAEMON_NAME)\" -w keydogger.c -o keydogger
 
 .PHONY: dependency-check
diff --git a/keydogger.c b/keydogger.c
index 70ba5ac..5136ca8 100644
--- a/keydogger.c
+++ b/keydogger.c
@@ -832,7 +832,7 @@ void daemonize_keydogger()
         exit(ERENAM);
     }
 
-    fd = open("/var/log/keydogger.log", O_RDWR | O_CREAT | O_APPEND);
+    fd = open("/var/log/keydogger.log", O_RDWR | O_CREAT | O_APPEND, 0600);
     if (fd < 0)
     {
         printf("Error opening %s\n", "/var/log/keydogger.log");

